master/doc/munin.conf.pod.in

```diff
@@ -204,6 +204,21 @@ Unfortunately, due to the limited functionality of the SSL module in use, it is
 not possible to provide finer-grained filtering.  By default this value is not
 defined.  Affects: munin-update.
 
+=item B<process_timeout_update> <value>
+
+This directive will set the maximum amount of time the munin-update task may run.
+Default value is 240 so we'll make sure the update ended within the 5 minutes
+timespan needed to have complete graphs without gaps.
+
+Do not set this value higher, unless all addresses are using ssh with munin-async.
+
+=item B<process_timeout_worker> <value>
+
+This directive will set the maximum amount of time the munin-update task may run
+1 single job. This value can't be bigger than process_timeout_update.
```

Comment:
Please clarify the meaning of *job* here. Probably it refers to a *node*?

Regarding the name of the setting: I would prefer a name, that does not rely on the internal mechanic used in munin (e.g. *worker*). Maybe `timeout_per_node` would be more suitable?